department-of-veterans-affairs/va.gov-cms
# VACMS-5339 vamc meta refactoring

## Description
Closes #5339

## Testing done
Visual / behat

## QA steps
- As an administrator, visit content type page for each vamc (outlined in task) and visually verify field_meta_title is not present.
- As an administrator, visit admin/config/search/metatag/node and verify that Page title pattern is set to `[node:title] | [node:field_office] | Veterans Affairs`


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
